webapp/channels/src/components/admin_console/schema_admin_settings.jsx

```diff
@@ -447,7 +447,7 @@ export default class SchemaAdminSettings extends React.PureComponent {
             inputType = 'textarea';
         }
 
-        let value = this.state[setting.key] || '';
+        let value = this.state[setting.key] ?? '';
```

Comment:
If we are doing optional chaining should we use “this.state?.[setting.key]?? ‘’”

Reply:
`this.state` is always defined, so we shouldn't need to. I should've also mentioned that this was the value to have these set to `0`